crates/polars-lazy/src/frame/mod.rs

```diff
@@ -1412,14 +1412,15 @@ impl LazyFrame {
     /// columns are considered.
     pub fn drop_nulls(self, subset: Option<Vec<Expr>>) -> LazyFrame {
         match subset {
-            None => self.filter(all_horizontal([col("*").is_not_null()])),
+            None => self.filter(all_horizontal([col("*").is_not_null()]).unwrap()),
```

Comment:
I don't think we should unwrap. There can be a dataframe without any columns. We have try_delayed for this.